app/src/androidTest/java/com/github/sdp_begreen/begreen/fragments/CameraFragmentTest.kt

```diff
@@ -60,7 +76,7 @@ class CameraFragmentTest {
         launchFragmentInContainer { fragment }
 
         // Click the add new post button
-        onView(withId(R.id.addNewPostBtn)).perform(ViewActions.click())
+        onView(withId(R.id.addNewPostBtn)).perform(click())
```

Comment:
Thanks for having removed all the `ViewActions`, it makes the code more readable :+1: